src/nginxconfig/i18n/zh-cn/templates/domain_sections/https.js

```diff
@@ -32,8 +32,8 @@ export default {
     enableHttp2Connections: `${common.enable} ${common.http}/2 连接`,
     http3: `${common.http}/3`,
     enableHttp3Connections: `${common.enable} ${common.http}/3 连接`,
-    portReuse: 'Reuseport', // TODO: translate
-    enableReuseOfPort: `${common.enable} reuseport to generate a listening socket per worker`, // TODO: translate
+    portReuse: '端口重用',
+    enableReuseOfPort: `${common.enable} <b>重用端口</b> 为每个 NGINX Worker 单独生成一个监听套接字`,
```

Comment:
```suggestion
    enableReuseOfPort: `${common.enable} 重用端口 为每个 NGINX Worker 单独生成一个监听套接字`,
```
We don't have raw HTML enabled for this string, and I'm not sure I see the need, we don't use bold for the English version of the string?